content/lib/pencil-tracer.js

```diff
@@ -243,6 +265,28 @@
       return args;
     };
 
+    CoffeeScriptInstrumenter.prototype.substringByLocation = function(location) {
+      var j, lineNum, ref, ref1, result;
+      result = "";
+      for (lineNum = j = ref = location.first_line, ref1 = location.last_line; ref <= ref1 ? j <= ref1 : j >= ref1; lineNum = ref <= ref1 ? ++j : --j) {
+        result += lineNum === location.first_line && lineNum === location.last_line ? this.lines[lineNum].slice(location.first_column, +location.last_column + 1 || 9e9) : lineNum === location.first_line ? this.lines[lineNum].slice(location.first_column) : lineNum === location.last_line ? this.lines[lineNum].slice(0, +location.last_column + 1 || 9e9) : this.lines[lineNum];
+      }
+      return result;
+    };
+
+    CoffeeScriptInstrumenter.prototype.argsToString = function(argNodes) {
+      var arg;
```

Comment:
Why is this scoped out here? Looks like it's only used as a temp inside the function you're returning. So declare it there, rather than here.